### DISCUSSION TITLE
Update SDK to EMF 2.40?

### DISCUSSION THREAD

**Discussion by iloveeclipse** [eclipse-platform/eclipse.platform.releng.aggregator#2375] [General]
Mostly the question to @merks : I see that Xtext nightly builds are using already EMF 2.40 artifacts, but SDK build doesn't.
Is there a plan to update SDK aggregator to latest EMF?

**Comment by merks** [comment]
I haven't built a milestone yet:

https://download.eclipse.org/modeling/emf/emf/builds/milestone/latest

I suppose Xtext is building against:

https://download.eclipse.org/modeling/emf/emf/builds/nightly/latest

But the platform only ever uses stable sites with stable (immutable) content.

Of course I plan create a milestone for M1 for SimRel and the Platform's TP.

If there is reason to do it sooner rather than later, let me know and I'll make it happen sooner...

**Comment by iloveeclipse** [comment]
I've suspected that, thanks for confirmation it is not an oversight.
I assume M1 will be in a week from now, so therefore no extra work needed, thanks Ed.

**Reply by merks** [reply]
FYI, I don't actually anticipate any changes between now and M1 for EMF.  Most of these changes were to eliminate warnings when compiling against Java 21 and I have no open issues I expect to address.   So I will try to get this done today.

**Comment by iloveeclipse** [comment]
Background: we have a weekly internal platform build that was failing with internal checks because we had Xtext nightly requiring higher EMF version as SDK nightly.

**Reply by merks** [reply]
This PR, when merged, will solve that problem:

https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/pull/2376
